docs/getting-started/cronos-testnet.md

```diff
@@ -31,6 +31,19 @@ The latest Crypto.org Chain Testnet has been named as **Cronos**.
 
 This is a detailed documentation for setting up a Validator or a full node on Crypto.org Cronos testnet `cronostestnet_338-3`.
 
+
+Before we start, please note that there was new binary upgrade:
+
+- Start the node with the older binary version v0.6.0;
+- Sync-up with the blockchain until it reaches the target upgrade block height `1553700`;
+- Update the binary to 0.7.0-rc1-testnet and update `~/.cronos/config/app.toml` [file link](https://github.com/crypto-org-chain/cronos-testnets/blob/main/cronostestnet_338-3/app.toml) (the best way to do so is to directly download the new [app.toml](https://github.com/crypto-org-chain/cronos-testnets/blob/main/cronostestnet_338-3/app.toml) and replace the whole local app.toml;
```

Comment:
Might consider using the raw form: 
https://raw.githubusercontent.com/crypto-org-chain/cronos-testnets/main/cronostestnet_338-3/app.toml